packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,342 @@
+      </View>
+    )
+  }
+
+  const { loading, error, result } = useAsync(
+    async () => {
+      const response = await fetch(dappsListUrl, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          Accept: 'application/json',
+        },
+      })
+      return response.json()
+    },
+    [],
+    {
+      onSuccess: (result) => {
+        Logger.debug(TAG, `fetch onSuccess: ${JSON.stringify(result)}`)
+        try {
+          const categoriesById: { [id: string]: CategoryWithItems } = {}
+          result.categories.forEach((cat: any) => {
+            categoriesById[cat.id] = {
+              id: cat.id,
+              name: cat.name,
+              fontColor: cat.fontColor,
+              backgroundColor: cat.backgroundColor,
+              items: [],
+            }
+          })
+          result.applications.forEach((app: any) => {
+            categoriesById[app.categoryId].items.push({
+              id: app.id,
+              name: app.name,
+              iconUrl: app.logoUrl,
+              description: app.description,
+              dappUrl: app.url,
+            })
+          })
+          setcategoriesWithItems(Object.values(categoriesById))
+        } catch (e) {
+          Logger.error(TAG, "Couldn't parse response", e as Error)
+        }
+      },
+    }
+  )
+
+  const onItemPress = (dapp: DappItem) => {
+    setDappSelected(dapp)
+    setBottomSheetVisible(true)
+  }
+
+  const onBottomSheetClose = () => {
+    setBottomSheetVisible(false)
+  }
+
+  const onHelpPress = () => {
+    setHelpDialogVisible(true)
+  }
+
+  const onNavigationButtonPress = () => {
+    if (!dappSelected) {
+      // Should never happen
+      Logger.warn(TAG, 'Internal error. There was no dapp selected')
+      return
+    }
+    navigateToURI(dappSelected.dappUrl)
+    setBottomSheetVisible(false)
+  }
+
+  return (
+    <SafeAreaView style={styles.safeAreaContainer}>
+      <CustomHeader
+        left={<BackButton />}
+        title={t('dappsScreen.title')}
+        right={
+          <TopBarIconButton
+            icon={<Help />}
+            onPress={onHelpPress}
+            style={styles.helpIconContainer}
+          />
+        }
+      />
+      <BottomSheet isVisible={isBottomSheetVisible} onBackgroundPress={onBottomSheetClose}>
+        <View>
+          <View style={styles.bottomSheetHeader}>
+            <TopBarIconButton icon={<QuitIcon />} onPress={onBottomSheetClose} />
+          </View>
+          <View style={styles.centerContainer}>
+            <Text style={{ ...fontStyles.h2, textAlign: 'center', paddingVertical: 16 }}>
+              {t('dappsScreenBottomSheet.title', { dappName: dappSelected?.name })}
+            </Text>
+            <Text style={{ ...fontStyles.regular, textAlign: 'center', flex: 1 }}>
+              {t('dappsScreenBottomSheet.message')}
+            </Text>
+
+            {/* <Text>Dont show again Message</Text> */}
```

Comment:
There is a discussion regarding if we want this feature or not: https://valora-app.slack.com/archives/C02TH9GNERL/p1642102112018200

Reply:
Lets delete this commented out stuff in the meantime, if we decide to add it we'll need to add extra logic anyways :)